pubspec.yaml

```diff
@@ -81,6 +80,7 @@ dependencies:
   wakelock: ^0.6.2
   xml2json: ^6.0.0
   yaml: ^3.1.0
+  extended_image: ^8.2.0
```

Comment:
Same here, I've got build errors for no fault of mine, reinstalling this dependency resolved the issue.